src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -39,7 +39,7 @@ const formMutators = {
     utils.changeValue(state, 'amount', () => args[0])
   },
   onTokenChange: (args, state, utils) => {
-    utils.changeValue(state, 'amount', () => '')
+    utils.changeValue(state, 'amount', () => state.formState.values.amount)
```

Comment:
what is this fix for?

Reply:
To avoid reset the input if the user puts a value and then selects a token